site/pages/pledge/[address].tsx

```diff
@@ -21,21 +36,20 @@ export const getStaticProps: GetStaticProps = async (ctx) => {
   }
 
   try {
-    const data = await getPledgeByAddress(address.toLowerCase());
+    const data = await getPledgeByAddress(resolvedAddress);
     if (!data) throw new Error("Not found");
 
     pledge = data;
   } catch (error) {
-    pledge = DEFAULT_VALUES;
+    pledge = { ...DEFAULT_VALUES, ownerAddress: resolvedAddress };
   }
 
   return {
     props: {
-      pageAddress: address.toLowerCase(),
-      pledge: {
-        ...pledge,
-        ownerAddress: pledge?.ownerAddress || address.toLowerCase(),
-      },
+      canonicalUrl: `${urls.pledges}/${address}`,
```

Comment:
I think I may have gave some bad advice a few weeks ago when thinking about the retirement urls

I originally thought we wanted to point search engines from `/pledge/0x123` to the .klima or .ens version, if it exists
But I don't think this is a good idea anymore, so we just do self-referential urls as you did here. Not really doing much afaik.

We should revisit at some point